Update Retail React App Page Designer integration README (#1041)
SalesforceCommerceCloud/pwa-kit: packages/template-retail-react-app/app/page-designer/README.md

        ____                      ____            _
       / __ \____ _____ ____     / __ \___  _____(_)___ _____  ___  _____
      / /_/ / __ `/ __ `/ _ \   / / / / _ \/ ___/ / __ `/ __ \/ _ \/ ___/
     / ____/ /_/ / /_/ /  __/  / /_/ /  __(__  ) / /_/ / / / /  __/ /
    /_/    \__,_/\__, /\___/  /_____/\___/____/_/\__, /_/ /_/\___/_/
                /____/                          /____/

---

This folder contains React components and utilities that render pages from [Page Designer](https://documentation.b2c.commercecloud.salesforce.com/DOC2/topic/com.demandware.dochelp/content/b2c_commerce/topics/page_designer/b2c_creating_pd_pages.html).

> It is living folder where you add React.js components responsible for rendering Page Designer components that have been serialized to JSON.

Reviewer: Did you mean to say "It is a living" .. 

Is "It" page designer? or the page designer folder?

Reviewer 2: I don't think "living folder" translates well to other cultures.

Reviewer: I agree. But I don't have any suggestions for alternative 🤷. We could be more direct and say something along the lines of "this is where you'll place all your page designer components that you develop"

> It is living folder where you add React.js components responsible for rendering Page Designer components that have been serialized to JSON.

Reviewer 2: ```suggestion
Use this folder to add React components that can render Page Designer components that have been serialized to JSON.
```

> It includes components for layout and visualization of images, grids and and carousels.

Reviewer 2: ```suggestion
This folder includes components for layout and visualization of images, grids, and carousels.
```

Reviewer 2: Do we want to describe these components as defaults?

> **By default, Page Designer integration is not enabled in the Retail React App.**

Reviewer 2: We should link to instructions on how to enable the integration.

## Folder Structure

> -   **`/core`** - Base components for rendering: `<Page>`, `<Region>`, and `<Component>`. You shouldn't need to modify them, but will use `<Page>` to render Page Designer content. The other components are useful when you are creating new assets.

Reviewer 2: ```suggestion
-   **`/core`** - Base components for rendering: `<Page>`, `<Region>`, and `<Component>`. Use `<Page>` to render Page Designer content. Use `<Region>` and `<Component>` for creating new assets.
```

> -   **`/assets`** - Non-visual components used in Page Designer. Includes `<Image>` and `<ImageWithText>` as well as other Page Designer assets you want to use in your PWA-Kit app. If you need to visualize a component, you would add it here.

Reviewer 2: ```suggestion
-   **`/assets`** - Non-visual components used in Page Designer. Includes `<Image>` and `<ImageWithText>` as well as any other Page Designer assets that you want to use in your PWA-Kit app. If you need to visualize a component, add it here.
```

-   **`/layouts`** - Components responsible for layout. Includes various grids and a `<Carousel>` component.

## Sample Usage

> First, create a new file, `app/pages/page-viewer/index.jsx`, and add the following:

Reviewer 2: ```suggestion
Create a new file called `app/pages/page-viewer/index.jsx`, and add the following:
```

export default PageViewer
```

> Second, modify `app/routes.jsx`, adding a route for `<PageViewer>`:

Reviewer 2: ```suggestion
Second, modify `app/routes.jsx`, adding a route for `<PageViewer>`:
```
```suggestion
Edit `app/routes.jsx` and add a route for `<PageViewer>`:
```

> Second, modify `app/routes.jsx`, adding a route for `<PageViewer>`:

Reviewer 2: ```suggestion
Open `app/routes.jsx` and add a route for `<PageViewer>`:
```
